### ISSUE TITLE
🧩 FEATURE REQUEST:  Expose a RSS URL to view your daily.dev feed in a 3rd-party RSS reader

### ISSUE THREAD

**Issue by Victor239** [dailydotdev/daily#996]
### Concise Description

Expose a RSS URL to view your daily.dev feed in a 3rd-party RSS reader

### Description

I use a RSS reader for many categories of news already - news in my country, my hobbies, YouTube feeds, podcasts, etc. Daily.dev would make another good RSS feed to add to my tech news category, and I find it nice I can customise it further on web app. 

### Solution Proposed

Expose a RSS feed URL for your user's feed.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @Victor239,

We have no plans at the moment to add support for RSS readers. It's technically challenging as we generate your feed only once you visit our platform, which may impact the order of what you see. And anyway, we focus on other initiatives. In the meantime, we offer RSS support for your bookmarks if you are interested.

**Comment by Victor239** [comment]
You could just set the RSS feed to update every X hours and generate the feed on those intervals. 

I have no bookmarks, I just found your site this week and am intrigued by it but if there's no RSS support then it's a shame because I won't visit it as I get all my other tech news via RSS already.

**Comment by idoshamun** [comment]
@Victor239 like everything the devil is in the details. As simple as your suggestion sounds, it won't work in many cases. RSS major assumption is that the feed is chronologically sorted, which is not the case here. So if we use push-based RSS systems every several hours, you may get spammed with tens of articles. Another challenge is that our recommendation system learns what you like or dislike based on your activity which is obviously not possible with a third-party software so our feed will not be able to improve itself for your liking. Anyway, I understand your point, given this is a very low priority for us I don't envision it happening given the required effort. I hope your understand

**Comment by ddahan** [comment]
Too bad it's so low priority, it's the only thing missing so that I use daily.dev.
Because the dream tool to me , is a mix of specific author articles I want to read + new articles to discover.
I understand the ratio cost/benefit is too bad however.
